next-app/app/page.js

```diff
@@ -138,41 +197,94 @@ function WeatherData({ apiData }) {
-                    <i className={weatherIcon} style={{ fontSize: '150px' }}></i>
+                  <div className="card text-center">
+                    <i className={`mb-0 ${weatherIcon}`} style={{ fontSize: '150px' }}></i>
                     <div className="card-body text-center">
-                      <div style={{ display: 'inline-block', textAlign: 'left' }}>
-                        <h5 className="mb-0">
+                      <div className="d-inline-block weather">
+                        <h5 className="text-center fs-3 mt-0 mb-0">
                           <b>{index === 0 ? 'Today' : day}</b>
                         </h5>
-                        <p className="card-title mt-0">
+                        <p className="card-title text-center mt-0 mb-0">
                           <small className="text-body-secondary">{apiDate}</small>
                         </p>
-                        <li className="card-text">Temperature: {value.temperature}°F</li>
-                        <li className="card-text">Humidity: {value.humidity}%</li>
-                        <li className="card-text">Wind: {value.wind} mph</li>
-                        <li className="card-text">Rain Chance: {value.precipitation}%</li>
-                        <li className="card-text">Weather Condition: {value.weathercode}</li>
+                        <p className="card-text text-center fs-3 mt-0 mb-0">
+                          <b>{value.temperature}°F</b>
+                        </p>
+                        <ListGroup variant="flush">
+                          <ListGroup.Item className="card-text">
+                            <div className="float-start">Humidity: </div>
+                            <div className="float-end">
+                              <b className="text-end">{value.humidity}%</b>
+                            </div>
+                          </ListGroup.Item>
+                          <ListGroup.Item className="card-text">
+                            <div className="float-start">Wind:</div>
+                            <div className="float-end">
+                              <b>{value.wind} mph</b>
+                            </div>
+                          </ListGroup.Item>
+                          <ListGroup.Item className="card-text">
+                            <div className="float-start">Rain Chance: </div>
+                            <div className="float-end">
+                              <b>{value.precipitation}%</b>
+                            </div>
+                          </ListGroup.Item>
+                          <ListGroup.Item className="card-text">
+                            <div className="float-start">Forecast: </div>
+                            <div className="float-end">
+                              <b>{value.weathercode}</b>
+                            </div>
+                          </ListGroup.Item>
+                        </ListGroup>
```

Comment:
maybe omit the colon delimiters, now that the values are right-aligned? just a stylistic choice, but I think it looks nicer (see Forecast)
> ![image](https://github.com/user-attachments/assets/76a227a4-f5ae-452a-b21a-b446134f8d56)